spec/models/rent_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe Rent do
+  subject(:rent) { build(:rent) }
+
+  it { is_expected.to validate_presence_of(:start_date) }
+  it { is_expected.to validate_presence_of(:end_date) }
+  it { is_expected.to validate_presence_of(:book) }
+  it { is_expected.to validate_presence_of(:user) }
+
+  describe '#save' do
+    context 'With valid params' do
+      it 'returns the new rent object' do
+        rent.save!
+        expect(rent).to be_present
+      end
+    end
+  end
+end
+
+describe Rent do
```

Comment:
This is fine in this case  because you have specific validations that cant be validated with the common  validations